content/docs/getting-started/installation/kubernetes/powermax/prerequisite/_index.md

```diff
@@ -224,69 +300,74 @@ Example:
+   These values can be obtained using base64 encoding as described in the
+   following example:
+
 ```bash
 echo -n "myusername" | base64
 echo -n "mypassword" | base64
 ```
-where *myusername* and *mypassword* are credentials for a user with vCenter privileges.
-3.
-4. Run following command to create the configmap
-  ```bash
-  kubectl create -f vcenter-secret.yaml
-  ```
->Note: Name of the secret should always be `vcenter-creds`.
 
-{{% /tab %}}
-{{< /tabpane >}}   
+where _myusername_ and _mypassword_ are credentials for a user with vCenter
+privileges. 3. 4. Run following command to create the configmap
 
-Choose your multipathing software between [Multipath](#linux-multipathing-requirements) & [PowerPath](#powerpath-for-linux-requirements)
+```bash
+kubectl create -f vcenter-secret.yaml
+```
 
-{{< tabpane text=true lang="en" >}}
-{{< tab header="Linux Multipathing" lang="en" >}}
-{{< markdownify >}}
+> Note: Name of the secret should always be `vcenter-creds`.
+
+{{% /tab %}} {{< /tabpane >}}
 
+Choose your multipathing software between
+[Multipath](#linux-multipathing-requirements) &
+[PowerPath](#powerpath-for-linux-requirements)
+
+{{< tabpane text=true lang="en" >}}
+{{< tab header="Linux Multipathing" lang="en" >}} {{< markdownify >}}
 
 ### Linux Multipathing Requirements
 
- Configure Linux multipathing before installing the CSI Driver.
+Configure Linux multipathing before installing the CSI Driver.
 
- **Supported Multipathing.** 
-  - Dell PowerMax supports Linux multipathing (DM-MPIO) and NVMe native multipathing.  
-  - Configure Linux multipathing before installing the CSI Driver.
+**Supported Multipathing.**
 
+- Dell PowerMax supports Linux multipathing (DM-MPIO) and NVMe native
+  multipathing.
+- Configure Linux multipathing before installing the CSI Driver.
 
 {{< /markdownify >}}
 
-
-{{< collapse id="1" title="NVMe" >}}For NVMe connectivity native NVMe multipathing is used.{{< /collapse >}}
-<br>
+{{< collapse id="1" title="NVMe" >}}For NVMe connectivity native NVMe
+multipathing is used.{{< /collapse >}} <br>
 {{< collapse id="2" title="FC/iSCSI" >}}
 
-1. Configuration steps: 
+1. Configuration steps:
 
    - Install the Device Mapper Multipathing package on all nodes:
-        -  `dnf install device-mapper-multipath`
-        -   `apt install multipath-tools`
+     - `dnf install device-mapper-multipath`
+     - `apt install multipath-tools`
    - Ensure the `mpathconf` command is available on all Kubernetes nodes.
    - Enable multipathing: `mpathconf --enable --with_multipathd y`
-   - Edit `/etc/multipath.conf` to enable `user_friendly_names` and `find_multipaths`.
-    
+   - Edit `/etc/multipath.conf` to enable `user_friendly_names` and
+     `find_multipaths`.
+
 <br>
 
-2. Best Practices 
+2.  Best Practices
 
-    Use these options in multipath.conf for efficient path detection:
+        Use these options in multipath.conf for efficient path detection:
```

Comment:
On the rendered page, this is displayed inside the code block with the text below. It should be above the block.